<launch>
  <node pkg="disparity_to_point_cloud" type="disparity_to_point_cloud_node" name="disparity_to_point_cloud" output="screen">
    <remap to="/fused_depth_0" from="/disparity"/>
    <remap to="/omi_cam/point_cloud" from="/point_cloud"/>
  </node>

  <!-- <node pkg="tf" type="static_transform_publisher" name="camera_tf_broadcaster" args="0 0 0 3.14 0 3.1415 stereo_cam::stereo_camera_link camera_optical_frame 10" /> -->
  <!-- <node pkg="tf" type="static_transform_publisher" name="camera_link_tf_broadcaster" args="0 0 0 0 0 0 world camera_optical_frame 10" /> -->
  <!-- the triplets are rotated 25 deg -->
  <node pkg="tf" type="static_transform_publisher" name="camera_tf_broadcaster" args="0 0 0 -1.57 0 -1.57 camera_link camera_optical_frame 10" />

  <!-- <node name="rviz" pkg="rviz" type="rviz" output="screen" args="-d $(find disparity_to_point_cloud)/rviz/d2pc.rviz" /> -->
</launch>
